tests/wamr-test-suites/test_wamr.sh

```diff
@@ -423,7 +431,7 @@ function spec_test()
 
     # multi-module only enable in interp mode
     if [[ 1 == ${ENABLE_MULTI_MODULE} ]]; then
-        if [[ $1 == 'classic-interp' || $1 == 'fast-interp' ]]; then
+        if [[ $1 != 'fast-jit' ]]; then
```

Comment:
Seems we should add `multi-tier-jit` also, which doesn't support multi-module:
`if [[ $1 != 'fast-jit' || $1 != 'multi-tier-jit' ]]; then`

Reply:
✅ changed the condition based on your advice